website/package.json

```diff
@@ -8,8 +8,8 @@
     "npm": ">=7.0.0"
   },
   "devDependencies": {
-    "@hashicorp/platform-cli": "^2.5.1",
-    "dart-linkcheck": "2.0.15",
```

Comment:
is the removal of `dart-linkcheck` intentional?

Reply:
👀 no it was not.